src/SingleBookingPage/bookingTypes/MulticityOverlayTrip.js

```diff
@@ -3,7 +3,7 @@
 import * as React from 'react';
 import { createFragmentContainer, graphql } from 'react-relay';
 
-import Multicity from './Multicity';
+import Multicity from './MulticityTrip';
```

Comment:
I think you forgot to rename the import itself and it's usages below 🙂

Reply:
good catch, fixed